docs/website/blog/2023-06-21-open-api-spec-for-dlt-init.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: open-api-spec-for-dlt-init
+title: "dlt & openAPI code generation: A step beyond APIs and towards 10,000s of live datasets"
+image: https://camo.githubusercontent.com/1aca1132999dde59bc5b274aeb4d01c79eab525941362491a534ddd8d1015dce/68747470733a2f2f63646e2e6c6f6f6d2e636f6d2f73657373696f6e732f7468756d626e61696c732f32383036623837336261316334653065613338326562336234666261663830382d776974682d706c61792e676966
+authors:
+  name: Matthaus Krzykowski
+  title: Co-Founder & CEO at dltHub
+  url: https://twitter.com/matthausk
+  image_url: https://pbs.twimg.com/profile_images/642282396751130624/9ixo0Opj_400x400.jpg
+tags: [fastapi, feature, openapi, dlt pipline, pokeapi]
+---
+Today we are releasing a proof of concept of the [`dlt init`](https://dlthub.com/docs/walkthroughs/create-a-pipeline) extension that can [generate `dlt` pipelines from an OpenAPI specification.](https://github.com/dlt-hub/dlt-init-openapi)
+
+If you build APIs, for example with [FastAPI](https://fastapi.tiangolo.com/), you can, thanks to the [OpenAPI spec,](https://spec.openapis.org/oas/v3.1.0) automatically generate a [python client](https://pypi.org/project/openapi-python-client/0.6.0a4/) and give it to your users. Our demo takes this a step further and enables you to generate advanced `dlt` pipelines that, in essence, convert your API into a live dataset.
+
+You can see how Marcin generates such a pipeline from the OpenAPI spec using the [Pokemon API](https://pokeapi.co/) in the Loom below.  
+[![marcin-demo](https://camo.githubusercontent.com/1aca1132999dde59bc5b274aeb4d01c79eab525941362491a534ddd8d1015dce/68747470733a2f2f63646e2e6c6f6f6d2e636f6d2f73657373696f6e732f7468756d626e61696c732f32383036623837336261316334653065613338326562336234666261663830382d776974682d706c61792e676966)](https://www.loom.com/share/2806b873ba1c4e0ea382eb3b4fbaf808?sid=501add8b-90a0-4734-9620-c6184d840995)  
+  
+Part of our vision is that each API will come with a `dlt` pipeline - like it now often comes with a python client. We believe that very often API users do not really want to deal with endpoints, http requests, and JSON responses. They need live, evolving datasets that they can place anywhere they want so that it's accessible to any workflow.
+
+We believe that API builders will bundle `dlt` pipelines with their APIs only if such a process is hassle free. And the answer to that is code generation and the reuse of information from the OpenAPI spec. 
```

Comment:
We believe that API builders will bundle `dlt` pipelines with their APIs only if such a process is hassle free. One answer to that is code generation and the reuse of information from the OpenAPI spec.